include/openssl/ssl.h

```diff
@@ -674,8 +672,7 @@ int (*SSL_CTX_sess_get_new_cb(SSL_CTX *ctx)) (struct ssl_st *ssl,
 void SSL_CTX_sess_set_remove_cb(SSL_CTX *ctx,
                                 void (*remove_session_cb) (struct ssl_ctx_st
```

Comment:
all those function typedefs, and there isn't one here? :)

Reply:
I get ithese kind of changes on my git repo, as draft for a next PR.